src/main/finder.js

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2015-2016 Yuya Ochiai
```

Comment:
Please put the file into src/browser/components. It's one of directories for renderer process of electron (Chromium part).

src/main/ is for the main process of electron (Node.js part).